### DISCUSSION TITLE
Is it possible to add more than 3 songs in 1 week?

### DISCUSSION THREAD

**Discussion by libracrumbs** [KadeArchive/Kade-Engine#1844] [Q&A] [answered]
Because I don't want to switch between weeks after every 3 song, is there a way to do it?

**Comment by SuxxorState** [comment, marked as answer]
it is indeed possible.
for the sake of things, ill assume you know how to decompile/compile the game and whatnot.
in StoryMenuState.hx (located in the source folder), go to lines 28-35. there you will see the story mode song list situated with their respective weeks. simply add another song at the end of the week you want to edit and save the file, build the game and there ya go!
just in case you dont know what im talking about, just edit one of these lines:
`["Bopeebo", "Fresh", "Dadbattle"],`
and add it like this:
`["Bopeebo", "Fresh", "Dadbattle", "Cool Song"],`
hope that helps

**Reply by libracrumbs** [reply]
tysm
